# goal definition
string motionType
mm_msgs/MobileRobotPose goalPose
---
#result definition
bool success
std_msgs/Duration executionTime
---
#feedback definition
int32 errorFlag
